--- CubeRoverTopologyAppAc.hpp.orig	2020-10-12 22:40:23.644599922 -0700
+++ CubeRoverTopologyAppAc.hpp	2020-10-12 22:40:57.353001315 -0700
@@ -16,15 +16,15 @@
 extern Fw::SimpleObjRegistry simpleReg;
 
 // Component declarations
-extern Drv::BlockDriverImpl blockDriver("blockDriver");
-extern Svc::RateGroupDriverImpl rateGroupDriver("rateGroupDriver");
-extern Svc::ActiveRateGroupImpl rateGroupLowFreq("rateGroupLowFreq");
-extern Svc::ActiveRateGroupImpl rateGroupMedFreq("rateGroupMedFreq");
-extern Svc::ActiveRateGroupImpl rateGroupHiFreq("rateGroupHiFreq");
-extern Svc::CubeRoverTimeImpl cubeRoverTime("cubeRoverTime");
-extern Svc::TlmChanImpl tlmChan("tlmChan");
-extern Svc::CommandDispatcherImpl cmdDispatcher("cmdDispatcher");
-extern Svc::GroundInterfaceImpl groundInterface("groundInterface");
+extern Drv::BlockDriverImpl blockDriver;
+extern Svc::RateGroupDriverImpl rateGroupDriver;
+extern Svc::ActiveRateGroupImpl rateGroupLowFreq;
+extern Svc::ActiveRateGroupImpl rateGroupMedFreq;
+extern Svc::ActiveRateGroupImpl rateGroupHiFreq;
+extern Svc::CubeRoverTimeImpl cubeRoverTime;
+extern Svc::TlmChanImpl tlmChan;
+extern Svc::CommandDispatcherImpl cmdDispatcher;
+extern Svc::GroundInterfaceComponentImpl groundInterface;
 
 
 
