--- CubeRoverTopologyAppAc.hpp.orig	2020-11-17 23:59:43.240242967 -0800
+++ CubeRoverTopologyAppAc.hpp	2020-11-18 00:00:19.815182645 -0800
@@ -18,17 +18,17 @@
 extern Fw::SimpleObjRegistry simpleReg;
 
 // Component declarations
-extern Drv::BlockDriverImpl blockDriver("blockDriver");
-extern Svc::RateGroupDriverImpl rateGroupDriver("rateGroupDriver");
-extern Svc::ActiveRateGroupImpl rateGroupLowFreq("rateGroupLowFreq");
-extern Svc::ActiveRateGroupImpl rateGroupMedFreq("rateGroupMedFreq");
-extern Svc::ActiveRateGroupImpl rateGroupHiFreq("rateGroupHiFreq");
-extern Svc::CubeRoverTimeImpl cubeRoverTime("cubeRoverTime");
-extern Svc::TlmChanImpl tlmChan("tlmChan");
-extern Svc::CommandDispatcherImpl cmdDispatcher("cmdDispatcher");
-extern CubeRover::GroundInterfaceImpl groundInterface("groundInterface");
-extern CubeRover::UdpReceiverImpl udpReceiver("udpReceiver");
-extern CubeRover::NetworkManagerImpl networkManager("networkManager");
+extern Drv::BlockDriverImpl blockDriver;
+extern Svc::RateGroupDriverImpl rateGroupDriver;
+extern Svc::ActiveRateGroupImpl rateGroupLowFreq;
+extern Svc::ActiveRateGroupImpl rateGroupMedFreq;
+extern Svc::ActiveRateGroupImpl rateGroupHiFreq;
+extern Svc::CubeRoverTimeImpl cubeRoverTime;
+extern Svc::TlmChanImpl tlmChan;
+extern Svc::CommandDispatcherImpl cmdDispatcher;
+extern CubeRover::GroundInterfaceComponentImpl groundInterface;
+extern CubeRover::UdpReceiverComponentImpl udpReceiver;
+extern CubeRover::NetworkManagerComponentImpl networkManager;
 
 
 
