--- CubeRoverTopologyAppAc.hpp.orig	2020-11-26 14:35:04.218021212 -0800
+++ CubeRoverTopologyAppAc.hpp	2020-11-26 14:35:23.777138860 -0800
@@ -21,20 +21,20 @@
 extern Fw::SimpleObjRegistry simpleReg;
 
 // Component declarations
-extern Drv::BlockDriverImpl blockDriver("blockDriver");
-extern Svc::RateGroupDriverImpl rateGroupDriver("rateGroupDriver");
-extern Svc::ActiveRateGroupImpl rateGroupLowFreq("rateGroupLowFreq");
-extern Svc::ActiveRateGroupImpl rateGroupMedFreq("rateGroupMedFreq");
-extern Svc::ActiveRateGroupImpl rateGroupHiFreq("rateGroupHiFreq");
-extern Svc::CubeRoverTimeImpl cubeRoverTime("cubeRoverTime");
-extern Svc::TlmChanImpl tlmChan("tlmChan");
-extern Svc::CommandDispatcherImpl cmdDispatcher("cmdDispatcher");
-extern CubeRover::GroundInterfaceImpl groundInterface("groundInterface");
-extern CubeRover::UdpInterfaceImpl udpInterface("udpInterface");
-extern CubeRover::NetworkManagerImpl networkManager("networkManager");
-extern Svc::ActiveLoggerImpl activeLogger("activeLogger");
-extern Svc::HealthImpl health("health");
-extern CubeRover::WatchDogInterfaceImpl watchDogInterface("watchDogInterface");
-extern CubeRover::CameraImpl camera("camera");
+extern Drv::BlockDriverImpl blockDriver;
+extern Svc::RateGroupDriverImpl rateGroupDriver;
+extern Svc::ActiveRateGroupImpl rateGroupLowFreq;
+extern Svc::ActiveRateGroupImpl rateGroupMedFreq;
+extern Svc::ActiveRateGroupImpl rateGroupHiFreq;
+extern Svc::CubeRoverTimeImpl cubeRoverTime;
+extern Svc::TlmChanImpl tlmChan;
+extern Svc::CommandDispatcherImpl cmdDispatcher;
+extern CubeRover::GroundInterfaceComponentImpl groundInterface;
+extern CubeRover::UdpInterfaceComponentImpl udpInterface;
+extern CubeRover::NetworkManagerComponentImpl networkManager;
+extern Svc::ActiveLoggerImpl activeLogger;
+extern Svc::HealthImpl health;
+extern CubeRover::WatchDogInterfaceComponentImpl watchDogInterface;
+extern CubeRover::CameraComponentImpl camera;
 
 
 
