--- CubeRoverTopologyAppAc.hpp.orig	2020-12-20 11:34:17.490796390 -0800
+++ CubeRoverTopologyAppAc.hpp	2020-12-20 11:42:26.840069860 -0800
@@ -25,24 +25,24 @@
 extern Fw::SimpleObjRegistry simpleReg;
 
 // Component declarations
-extern Drv::BlockDriverImpl blockDriver("blockDriver");
-extern Svc::RateGroupDriverImpl rateGroupDriver("rateGroupDriver");
-extern Svc::ActiveRateGroupImpl rateGroupLowFreq("rateGroupLowFreq");
-extern Svc::ActiveRateGroupImpl rateGroupMedFreq("rateGroupMedFreq");
-extern Svc::ActiveRateGroupImpl rateGroupHiFreq("rateGroupHiFreq");
-extern Svc::CubeRoverTimeImpl cubeRoverTime("cubeRoverTime");
-extern Svc::TlmChanImpl tlmChan("tlmChan");
-extern Svc::CommandDispatcherImpl cmdDispatcher("cmdDispatcher");
-extern CubeRover::GroundInterfaceImpl groundInterface("groundInterface");
-extern CubeRover::UdpInterfaceImpl udpInterface("udpInterface");
-extern CubeRover::NetworkManagerImpl networkManager("networkManager");
-extern Svc::ActiveLoggerImpl activeLogger("activeLogger");
-extern Svc::HealthImpl health("health");
-extern CubeRover::NavigationImpl navigation("navigation");
-extern CubeRover::MotorControlImpl motorControl("motorControl");
-extern CubeRover::IMUImpl IMU("IMU");
-extern CubeRover::WatchDogInterfaceImpl watchDogInterface("watchDogInterface");
-extern CubeRover::CameraImpl camera("camera");
-extern CubeRover::ComLoggerImpl comLogger("comLogger");
+extern Drv::BlockDriverImpl blockDriver;
+extern Svc::RateGroupDriverImpl rateGroupDriver;
+extern Svc::ActiveRateGroupImpl rateGroupLowFreq;
+extern Svc::ActiveRateGroupImpl rateGroupMedFreq;
+extern Svc::ActiveRateGroupImpl rateGroupHiFreq;
+extern Svc::CubeRoverTimeImpl cubeRoverTime;
+extern Svc::TlmChanImpl tlmChan;
+extern Svc::CommandDispatcherImpl cmdDispatcher;
+extern CubeRover::GroundInterfaceComponentImpl groundInterface;
+extern CubeRover::UdpInterfaceComponentImpl udpInterface;
+extern CubeRover::NetworkManagerComponentImpl networkManager;
+extern Svc::ActiveLoggerImpl activeLogger;
+extern Svc::HealthImpl health;
+extern CubeRover::NavigationComponentImpl navigation;
+extern CubeRover::MotorControlComponentImpl motorControl;
+extern CubeRover::IMUComponentImpl IMU;
+extern CubeRover::WatchDogInterfaceComponentImpl watchDogInterface;
+extern CubeRover::CameraComponentImpl camera;
+extern CubeRover::ComLogger comLogger;
 
 
