--- CubeRoverTopologyAppAc.hpp.orig	2020-11-18 14:35:00.915941054 -0500
+++ CubeRoverTopologyAppAc.hpp	2020-11-18 14:35:47.859857878 -0500
@@ -18,17 +18,17 @@
 extern Fw::SimpleObjRegistry simpleReg;
 
 // Component declarations
-extern Drv::BlockDriverImpl blockDriver("blockDriver");
-extern Svc::RateGroupDriverImpl rateGroupDriver("rateGroupDriver");
-extern Svc::ActiveRateGroupImpl rateGroupLowFreq("rateGroupLowFreq");
-extern Svc::ActiveRateGroupImpl rateGroupMedFreq("rateGroupMedFreq");
-extern Svc::ActiveRateGroupImpl rateGroupHiFreq("rateGroupHiFreq");
-extern Svc::CubeRoverTimeImpl cubeRoverTime("cubeRoverTime");
-extern Svc::TlmChanImpl tlmChan("tlmChan");
-extern Svc::CommandDispatcherImpl cmdDispatcher("cmdDispatcher");
-extern Svc::ActiveLoggerImpl activeLogger("activeLogger");
-extern Svc::HealthImpl health("health");
-extern CubeRover::WatchDogInterfaceImpl watchDogInterface("watchDogInterface");
+extern Drv::BlockDriverImpl blockDriver;
+extern Svc::RateGroupDriverImpl rateGroupDriver;
+extern Svc::ActiveRateGroupImpl rateGroupLowFreq;
+extern Svc::ActiveRateGroupImpl rateGroupMedFreq;
+extern Svc::ActiveRateGroupImpl rateGroupHiFreq;
+extern Svc::CubeRoverTimeImpl cubeRoverTime;
+extern Svc::TlmChanImpl tlmChan;
+extern Svc::CommandDispatcherImpl cmdDispatcher;
+extern Svc::ActiveLoggerImpl activeLogger;
+extern Svc::HealthImpl health;
+extern CubeRover::WatchDogInterfaceComponentImpl watchDogInterface;
 
 
 
