--- /home/pseudolunar-relay/justin-ccs9-workspace/dma-dev-cuberoverpackage/Apps/FlightSoftware/PrimaryFlightController/FlightMCU/HAL/source/adc.c	2021-04-02 03:38:33.080477016 -0400
+++ source/adc.c	2021-04-06 01:54:24.477971640 -0400
@@ -384,7 +384,7 @@
     0x00000000U |
     0x00000000U |
     0x00000000U |
-    0x00000000U,
+    0x0000003FU,
     0x00000000U |
     0x00000000U |
     0x00000000U |
